Gems/LmbrCentral/Code/Source/Shape/EditorSplineComponentMode.cpp

```diff
@@ -35,6 +36,40 @@ namespace LmbrCentral
         m_vertexSelection.Destroy();
     }
 
+    void EditorSplineComponentMode::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<EditorSplineComponentMode, EditorBaseComponentMode>(AZ::Internal::NullFactory::GetInstance());
```

Comment:
update: Not sure what `AZ::Internal::NullFactory::GetInstance()` is for but it would be hard for a user to know to use this, maybe we should make a templated helper to raise the abstraction level of this?